### PR TITLE
catalog

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_catalog/)
- [TEST REPORT LINK](https://ISRLv2021.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    text-transform: uppercase;
+
+    background-color: $blueAccent;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      color: $blueAccent;
+      background-color: $whiteColor;
+      border-color: $blueAccent;
```
Comment (ZUBOV-ILLIA) [inline src/styles/blocks/card.scss:98]: `&__btn-link` must have `transition` option.

File: src/styles/blocks/navigation.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    font-size: $commonFontSize;
+    font-weight: 500;
+    color: $baseColor;
+    line-height: 18px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      color: $blueAccent;
+    }
```
Comment (ZUBOV-ILLIA) [inline src/styles/blocks/navigation.scss:33]: `&__link` must have `transition` option.

**Review by ZUBOV-ILLIA — CHANGES_REQUESTED** [review]

File: src/styles/utils/_loops.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@for $starsCount from 1 through 5 {
+  .stars--#{$starsCount} :nth-child(-n + #{$starsCount}) {
+    background: url(../../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```
[the remark this answers is not in the document]
Comment (ISRLv2021) [reply src/styles/utils/_loops.scss:5]: got it

File: src/styles/blocks/card.scss (hunk 1 above)
Comment (ISRLv2021) [reply src/styles/blocks/card.scss:98]: corrected

File: src/styles/blocks/navigation.scss (hunk 2 above)
Comment (ISRLv2021) [reply src/styles/blocks/navigation.scss:33]: Thanks a lot, completed

File: src/index.html [hunk 4]
```diff
@@ -4,9 +4,415 @@
...
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <a
+        href="#"
+        class="logo header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="logo__img">
```
Comment (Denchepornyuk) [inline src/index.html:20]:
```suggestion
          class="logo__img"
        >
```

File: src/index.html [hunk 5]
```diff
@@ -4,9 +4,415 @@
...
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover">
```
Comment (Denchepornyuk) [inline src/index.html:43]: fix closing quote in all places
```suggestion
              data-qa="nav-hover"
            >
```

File: src/index.html [hunk 6]
```diff
@@ -4,9 +4,415 @@
...
+              href="#gadgets">
+                гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#tablets">
+                планшеты
+            </a>
```
Comment (Denchepornyuk) [inline src/index.html:60]:
```suggestion
              href="#tablets"
            >
              планшеты
            </a>
```

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
Screenshot 2022-04-11 at 14 33 30

something wrong with stars

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
Screenshot 2022-04-11 at 17 41 25
the stars are still broken

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**